### ISSUE TITLE
Add publisher/journal: The Palaeontological Association

### ISSUE THREAD

**Issue by mark02139** [CDL-Dryad/dryad-product-roadmap#2917]
**Name to appear on membership page:** 
The Palaeontological Association is currently listed on the membership page, but misspelled (The **Paleontological** Association rather than The **Palaeontological** Association)

**Payment plan:**
PALASS has signed a new member agreement. Their 2 journals will be on the tiered plan **under Wiley, commencing Jan 1, 2024.**

**Journals included:** 
Palaeontology
Papers in Palaeontology

**Email contacts to receive submission notices: (note whether they receive all notices, or different lists for different journals)** 
n/a

**Email contacts to receive publication notices: (note whether they receive all notices, or different lists for different journals)** 
n/a

**ORCIDs of users who need access to admin dashboard: (they must log in to Dryad to set up their account before we can give them access, note whether each user administers a subset of journals)**
n/a

**Email contacts to discuss integration of journal metadata with Dryad:**
n/a

Development tasks:
- [x] add member name to membership page
- [x] update payment settings
- [x] update journal notification settings
- [x] add journal administrators
- [x] start integration process (track on new ticket)

**Comment by mark02139** [comment]
@mariaguerreiro

**Comment by ryscher** [comment]
- Changed payment type from SUBSCRIPTION to TIERED
- Moved The Palaeontological Association under the Wiley sponsorship
- Updated spelling of the association's name on the membership page
